### DISCUSSION TITLE
Cooler Master MasterFan MF120 Halo Do Not Stop

### DISCUSSION THREAD

**Discussion by Y4m4k** [Rem0o/FanControl.Releases#687] [Q&A]
Hi - I am unable to stop my Cooler Master MasterFan in Fan Control. When I was using AI Suite3, I could stop them. Any idea what I need to do so that they are able to stop? I tried doing a detect stop automatically, but it says fan never stopped. Rad Fans/Back Exhaust (both are Cooler Master MasterFan, back exhaust is a MF140) have the same issue. I am able to stop my front fans (they are Noctua NF-A14 iPPC-2000 IP67 PWM).

I appreciate any guidance!

![image](https://user-images.githubusercontent.com/86619370/165401418-0e2f5b15-8672-4515-b5fc-41a7bc1173da.png)

**Comment by Tomasoo7** [comment]
Hi, I have the same issue with Prisma aRGB AL12 Fans..

**Comment by Eeshen** [comment]
Same issue on some of my fans using ROG Strix MB

**Comment by Y4m4k** [comment]
Try this closed item —> https://github.com/Rem0o/FanControl.Releases/issues/704

**Reply by Eeshen** [reply]
That definitely took care of another issue I was having, during speed detection the fans would not go to zero.  
However, as I mentioned in another post, my fans continue to cycle between zero RPM and the set speed of what is FAN Control.  For example, as I watch my fans, they keep jumping between 0 RPM and the Flat Curve value I have.

The moment I change the fans to not follow Fan Control they go back to the fan speed as set by Qfan in the BIOS.

Another weird item is the GPU Fan widget is behaving exactly how the fans attached to the CPU pin are operating.  However, adjust the GPU Fan does not change any fan on the GPU or in the case itself.

ROG Strix MB

**Reply by Rem0o** [reply]
Many, many different things could be doing this.

1) First, make sure the % you set is one your fan can maintain.  If it's too low, it could trigger the cycling.
2) Make sure your start% and stop% set are good. If you are not sure, set both values to 0%.
3) Make sure Q-Fan/BIOS does NOT interfere with FanControl. Any "smart" mode in the bios must be turned off, as it will yank the steering wheel away from FanControl anytime it tries to take over.

**Reply by Eeshen** [reply]
#1.  I confirmed.
#2.  Not sure where I do this at.
#3.  I have no idea where to look for this possible smart mode, I've made so many changes in Bios and can't seem to find what to turn off.  I have disabled all Monitoring options other than for CPU (doing this makes fan run 100%).  But my fans continue to cycle.

Thank you for your suggestions.

**Reply by Rem0o** [reply]
for 2) 
![image](https://user-images.githubusercontent.com/14118956/169576985-984f317b-a221-4e19-bcba-465ad8c39f17.png)

**Reply by Eeshen** [reply]
I dont use any of those types of controls.  I just use Curve controls for my fans and set the minimum level to the level at which they will first turn on.

**Comment by Eeshen** [comment]
![image](https://user-images.githubusercontent.com/105397583/169577531-87288a49-ac2f-4353-a435-3bd7f9f579b9.png)

**Reply by Eeshen** [reply]
Here is a moment where, even though the fans should be running at 40%, they dropped to zero for a brief moment before ramping back up.

**Comment by Y4m4k** [comment]
Hit the carrot arrow next to RPM, make sure start% and stop% arent the cause. I had an issue with that as well. Set them to 0 fixed it.

**Reply by Eeshen** [reply]
I set both values to zero, now all the fans listed above cycle between the Low Speed setting of 40% and completely stop.....go back to 40%, then completely stop.

**Reply by Eeshen** [reply]
For additional information, I have these fans connected to Chassis 2, M.2 and CPU fan headers on the MB.

**Reply by Rem0o** [reply]
What if you set that minimum to let say 70%, do they hold their value?

**Reply by Y4m4k** [reply]
That means that the 40% is too low of percent for sustained spin, see at what percent the fans continue to spin. That would be considered the min the fan can spin at.

**Reply by Eeshen** [reply]
Minimum as "Stop%"  ???     All fans begin spinning at/around 30%.   If I switch them to follow the Max curve of 100%, they will go to 100% before dropping to zero and starting that cycle.

**Reply by Y4m4k** [reply]
From a trouble shooting perspective, on your fan curve. Leave Stop% at 0 to take away that variable.

**Reply by Rem0o** [reply]
@Eeshen  you'll have to go into your bios and figure the settings out a bit. PWM vs DC for your headers, any curves set in there etc.... My best guess is that "some" setting in there is messing stuff.

**Reply by Y4m4k** [reply]
What I would do, set manual fan curve and just move the slider from 0 and see where the fan can sustain spin.

**Reply by Eeshen** [reply]
Yup, I was actually already doing that.   Currently, have every fan set to Stop % 0 and set to run at 100%.   All fans go to 100% for a second or two (the duration is random) and then drop to zero (for a random duration 1-5 secs) then go back 100%......sometimes they dont even make it to 100% before it goes back to zero again, sometimes they do.   Very random.

**Reply by Y4m4k** [reply]
Or just follow @Rem0o directive - he is the developer 🙇‍♂️

**Reply by Y4m4k** [reply]
Yea, that sounds like a BIOS issue. You have a Asus MB? I think is the DC vs PWM settings in there. Had to move mine to DC when I was having problems.

**Reply by Eeshen** [reply]
I have mine all set to DC and set to flat curve of 50, except for the CPU which cannot be set to anything less than 60 flat.  
As I write this, my fans continue to cycle between full on (I have them set to Max fan curve in Fan Control) and complete off.

**Reply by Y4m4k** [reply]
Is it Asus? Make sure smart fan option is off.

**Comment by Eeshen** [comment]
Not sure where that option is.

I moved the fans (Top Fans) set on the CPU header to the AIO header and moved the AIO which was on the AIO header to the CPU header and now all my fans have stopped cycling.....all running at the Max flat setting....no cycling for a good minute now.

**Reply by Eeshen** [reply]
Thoughts on what is going on?

**Reply by Eeshen** [reply]
Nevermind, I spoke too soon.  They were running fine, until I went through the Assisted Setup again, now the same cycling is happening.

**Reply by Y4m4k** [reply]
This is how it is for me in Asus CrossHair VIII Dark Hero

First Enter BIOS in EZMode and go to QFan Control
![Inked220520134521_LI](https://user-images.githubusercontent.com/86619370/169584536-bfdf33d8-7739-4d4b-8246-7db71cadeb37.jpg)

Check header, ensure it is DC and Fan Off is checked.
![Inked220520134544_LI](https://user-images.githubusercontent.com/86619370/169584669-9e135ece-6ebf-4607-8740-4e148af9b4ec.jpg)

**Reply by Eeshen** [reply]
Yeah, I did this already.  Even tried setting the curve to flat at various levels.

**Reply by Y4m4k** [reply]
That how I got mine to work. There seems to be a conflict between BIOS and Fan Control in your case. I'll see if I can share anything else that is helpful.

**Reply by Eeshen** [reply]
Thank you, I sooo want to use Fan Control.  I can see the potential if I can get it to work.  :)

**Reply by Y4m4k** [reply]
Do you have any other fan software installed? AI Suite 3 if Asus MB?

**Reply by Eeshen** [reply]
Only thing that is installed that MIGHT be related is ArmouryCrate and Corsair ICue, but neither are actively running.  I uninstalled ArmouryCrate and nothing changed.  I dont see any fan control options except for the water pump in Icue

**Reply by Eeshen** [reply]
In my opinion, it is definitely related to the BIOS, as the BIOS Qfan settings seem to affect the cycling.

**Reply by Y4m4k** [reply]
Attach some pics of BIOs settings. Insert a USB stick and hit F12 in BIOS to take a screen grab.

**Reply by Eeshen** [reply]
Anything specific you are looking for?

**Reply by Y4m4k** [reply]
Just your fan BIOS settings maybe?

**Comment by Eeshen** [comment]
![220521130934](https://user-images.githubusercontent.com/105397583/169662278-12b74290-8359-482b-8a7d-f9cb44a956e9.JPEG)
![220521130854](https://user-images.githubusercontent.com/105397583/169662281-0493d95a-a08b-4edc-a688-9df70737f1ee.JPEG)
![220521130833](https://user-images.githubusercontent.com/105397583/169662285-b86c0735-5cd7-48fe-93ea-8aeccfd3790d.JPEG)
![220521130823](https://user-images.githubusercontent.com/105397583/169662291-e87f000d-81c4-4d99-86f9-ef046e4c3772.JPEG)
![220521130811](https://user-images.githubusercontent.com/105397583/169662295-5a26757e-e131-4b0f-b807-dc94d4b45dd9.JPEG)

**Reply by Eeshen** [reply]
I also have all the "Monitoring" options off in BIOS except for CPU.

**Reply by Y4m4k** [reply]
I’ll have a look. The BIOS is the same mine.

**Reply by Eeshen** [reply]
I mentioned it previously, but I want to mention it again, does it matter that I have three fans per MB header?  The total current draw is less than the MB header rating (they also run fine when Fan Control is not controlling them).

**Reply by Y4m4k** [reply]
CPU fan is set to PWM. Are the three fans connected to the CPU header? Shouldn’t matter, I have three 140mm fans connected to one header, however, it is a high amp header. It is working fine.

**Reply by Eeshen** [reply]
I would assume if there was an issue with the current draw, that the issue would occur when the BIOS is running the fans and not Fan Control, but it does not.  With only the BIOS, I can run them at any set speed in BIOS with no cycling.

**Reply by Y4m4k** [reply]
I don’t think it a current draw problem. You saying monitoring is off for everything except CPU, is the AIO linked to the CPU temp or do you have any fluid temp monitoring?

**Reply by Eeshen** [reply]
I just have the 1-wire connected to the AIO header.  The AIO fan speed is set by Icue (then Icue is closed).  All monitoring is off, cept to CPU because fans go full speed without it.

**Reply by Y4m4k** [reply]
What about BIOS --> Monitor Tab --> Scroll to the bottom --> Q-Fan Configuration. I have mine set to DC mode and Standard Profile except for the High Amp Fan Q-Fan Control.

![220522142919](https://user-images.githubusercontent.com/86619370/169710424-8e3e2713-24f9-4123-8264-164035e643a0.jpeg)

![220522142948](https://user-images.githubusercontent.com/86619370/169710441-87c05317-e1ae-44c9-ae23-6ce466bb5943.jpeg)

![220522142955](https://user-images.githubusercontent.com/86619370/169710450-e346ef1d-eb88-4389-a771-f4170b3ba470.jpeg)

**Reply by Eeshen** [reply]
Can you also take a screenshot of your Monitor (BIOS) page.  Should be just one screen.

**Reply by Eeshen** [reply]
So far, just looking at the fans on the CPU header, if I set them to 80% in Fan Control, they run for about 1-2 secs at 80% then drop to zero and slowly move back up before dropping to zero again, sometimes not even making it to 80% before dropping to zero again.....the same old cycling.  However, the cycling is not the same each time.

**Reply by Eeshen** [reply]
I have a 22Mb .mov clip to show you, but the attachments are limited to 10Mb.

**Reply by Y4m4k** [reply]
You can upload to Google Drive or Dropbox and share the link.

Here is the first half of the Monitor page from the BIOS (is this what you are looking for?). I have CPU temps ignored because I have a custom loop and track water temps.

Chassis 2 are my 3x 120mm radiator fans - linked to avg. water in temp
Chassis 3 is 1x 140mm back exhaust - linked to GPU temp
High Amp Fans are my 3x 140mm intake fans - linked to avg. water in temp.
W_Pump+ is my Res/Pump Combo - set at a flat 65%  

![220522224429](https://user-images.githubusercontent.com/86619370/169733764-ca2922a8-bc86-445a-9a66-4a1e370760c7.jpeg)

**Reply by Y4m4k** [reply]
Also, have you tried starting fresh with Fan Control? Going to the wrench icon next to Controls and doing an assisted setup again?

![image](https://user-images.githubusercontent.com/86619370/169735167-a4184503-8832-43ea-94ca-f7d4007c1078.png)

**Reply by Eeshen** [reply]
Yes, MANY times!

**Reply by Eeshen** [reply]
So weird, how yours is working but mine is not.  😢

**Reply by Y4m4k** [reply]
Do they ramp up then down with the assisted setup as the % go up and own, or are they also acting weird there too?

I remember you mentioning Icue for control some fans, then closing it. It could be that there is some background tasks running even those that application is closed. Maybe try deleting it totally, restart and then trying Fan Control. I don't have any fan control software other than "Fan Control". Once I installed and tested it out, I deleted AI Suite 3. Not really sure what else is could be.

**Reply by Y4m4k** [reply]
Also, try this. Maybe connect just one fan at a time, might help identify if there is a particular fan or set of fans that is/are causing the issue.

**Comment by Eeshen** [comment]
![image](https://user-images.githubusercontent.com/105397583/169816171-a73cdb7e-7292-45d4-bf00-7eeeef093dcd.png)

**Reply by Eeshen** [reply]
So while running "Identify Fan" all fans come to a complete stop and stay that way.  While determining each fan starting point, only those fans on CHAS 1 respond, the rest do not.  However, on the next step, determining fan stopping point, all fans immediately go to full speed and start ramping down.
While fans are going through this testing procedure, **the fans that run**, never go through any cycling.  The fans step through each speed %.

The cycling doesn't start until I reach the "Identify each fan" step (where I have sliders to adjust the fans).

Also, as you can see from the screenshot, I have Fan Control #6 and #7, I can't find what they do, the GPU Fan also does nothing. Any ideas?

**Comment by Eeshen** [comment]
Also, I have a Corsair Commander Pro I could attach the fans to.  Can Fan Control work "through" Commander Pro?  My fans are currently not connected to the Pro except for RBG.

Another thing I just noticed this morning.  My new fans (2x 90mm, Arctic Fans) appear to be holding at Max speed via Fan Control and not cycling.  However, the 6 Corsair QL fans and the exhaust fans for the Corsair AIO, cycling after completing the Identify each fan.

It is starting to feel like it is the Corsair fans and having 3 fans in parallel to each header might be an issue??

**Reply by Y4m4k** [reply]
I would disconnect the 3 fans connected to the header and just connect one and see if the issue persists.

Cant comment on the Commander Pro, dont have one.

There is some conflict happening in your system we are struggling to identify.

**Comment by Eeshen** [comment]
Why can Fan Control control the speed of the fans (at least when determining stop speed) during the Identify Fan procedure, but stops working once the procedure is finished?

Quick Edit:  Just ran Assisted Setup procedure again, all fans stopped, started during "fan start determination" and went to max and ramped down during stop fan determination.  Essentially, this procedure, worked as programmed.

However, the moment I get to "Identify Fans" everything goes to crap.

So it appears Fan Control and can control the fans during Assisted Setup, but loses that control once it reaches Identify Fans.


I will try connecting only one fan and see how it goes.

**Reply by Y4m4k** [reply]
Yea, exactly. That is the odd thing. That is why I think there is some conflict going on.

**Comment by Eeshen** [comment]
As a reminder, I have 3 fans connected with a parallel jumper to each header.....

-I ran Assisted Setup with only one fan connected each of set of fans (one fan per header).   After setup, the fans ran no problem at 100%, no cycling.  Fan Control operated well.
-While the one fan was running, I added a second fan to each set of fans.  Three sets of fans running, each set only having 2 fans connected.  No cycling, Fan Control ran the fans at 100% as I directed.
-Added third fan to each sets of fans, the cycling begins.  Even at 35% fan speed, each set of fans would cycle....with the on times being a random duration before it would drop out (0 RPM) then going back to 35%.   If I would unhook the third fan, Fan Control could establish control again.

Therefore, I would conclude that something is going on when having three fans connected to a same header.  The BIOS can run three fans with zero issues, so not a current draw issue.  But something with three fans, Fan Control, I assume, does not like.

**Reply by Y4m4k** [reply]
Sounds sensible, probably a bug with Fan Control.

**Reply by Rem0o** [reply]
The fact it starts acting up with 3 fans is really weird, but also telling. To me it seems like you are at the limit of the header. Don't know what the difference between FanControl or the BIOS directing the header would be where it would work great with BIOS and not FanControl.

**Reply by Eeshen** [reply]
So sad...
I've been all day with Fan Control and 2 fans per header and had zero problems.   Don't know what is so special with adding a third fan and as Rem0o said, the difference between the BIOS controlling them and Fan Control.

**Comment by Eeshen** [comment]
What options are there out there that I have to control nine fans or is there some adjustment that can be done in Fan Control ?

**Reply by Y4m4k** [reply]
I think we exhausted our options, unless someone else has any options.

**Comment by Eeshen** [comment]
Thanks for all your help and the extensive time you gave in trying to help me, Y4m4k

**Reply by Y4m4k** [reply]
No problem @Eeshen, happy to help. I am sorry we couldn't find a solution for you :pensive:. If you do make any progress, please update us!
